media/server/ipc/source/SessionManagementServer.cpp

```diff
@@ -82,6 +88,56 @@ bool SessionManagementServer::initialize(const std::string &socketName, unsigned
     {
         RIALTO_SERVER_LOG_SYS_WARN(errno, "Failed to change the permissions on the IPC socket");
     }
+
```

Comment:
This function became very long, I would prefer to have 2 functions here, uid_t getOwnerId(...) and gid_t getGroupId(...). chown at the end can stay where it is. You can place common constants (noOwnerChange, noGroupChange) in empty namespace at the top of this file (please look at other files as an example)

Reply:
it looks perfect now! :)